scss/_patterns_code-copyable.scss

```diff
@@ -3,59 +3,49 @@
 $cc-button-size: 36px;
 
 @mixin vf-p-code-copyable {
-  .p-code-copyable {
-    @extend %bordered-text-vertical-padding;
-    @extend %vf-bg--x-light;
-    @extend %vf-has-round-corners;
-    @extend %vf-is-bordered;
+  @include deprecate('3.0.0', 'p-code-copyable is being removed, use the code snippet block instead.') {
+    .p-code-copyable {
+      background-color: $color-code-background;
+      display: flex;
+      margin-bottom: $input-margin-bottom;
+      overflow: hidden;
+      padding: $spv-inner--x-small--scaleable $sph-inner;
+      position: relative;
+
+      & + & {
+        margin-top: 0; // overrides p + p
+      }
 
-    display: flex;
-    margin-bottom: $input-margin-bottom;
-    overflow: hidden;
-    position: relative;
+      &::before {
```

Comment:
I feel that we should approach this similarly to code numbered and extract it to common base styles.

Any tweak to icon positioning or something will have to be applied in both places, we need to share these styles as much as possible so they are consistent and up to date in both places.

Reply:
@bartaz updated